docs/reference/setup/install/brew.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[brew]]
+=== Install {es} on macOS with Homebrew
+
+Elastic publishes Homebrew formulae so you can install {es} with the
+https://brew.sh/[Homebrew] package manager.
+
+To install with Homebrew, you first need to tap the
+Elastic Homebrew repository:
+
+[source,sh]
+-------------------------
+brew tap elastic/tap
+-------------------------
+
+Once you've tapped the Elastic Homebrew repo, you can use `brew install` to
+install {es}:
+
+[source,sh]
+-------------------------
+brew install elastic/tap/elasticsearch
+-------------------------
+
+This installs the most recently released default distribution of {es}.
+Append the version number to install a previous version, for example
+`elastic/tap/elasticsearch@6.8`. To install the OSS distribution,
```

Comment:
Should this just say `brew install elasticsearch@6.8`. I apologize for my ignorance (I don't use brew often), but the command you show results in:

Error: No available formula with the name "elastic/tap/elasticsearch@6.8"

Reply:
You have to specify the elastic tap. The example is bogus, because the 6.8 version isn't available.